sympy/codegen/array_utils.py

```diff
@@ -768,6 +768,9 @@ def parse_indexed_expression(expr, first_indices=[]):
 
     >>> from sympy.codegen.array_utils import parse_indexed_expression
     >>> from sympy import MatrixSymbol, Sum, symbols
+    >>> from sympy.combinatorics import Permutation, PermutationGroup
+    >>> Permutation.print_cyclic = True
+
```

Comment:
Permutations are printed also elsewhere, in `nest_permutation`, for example.

Reply:
I added `print_cyclic=False` to every docstring in the module. Only `nest_permutation` and `parse_indexed_expression` failed. I've just pushed a new version that adds `print_cyclic=True` to those two doctests.